pkg/convert/pot.go

```diff
@@ -73,46 +53,175 @@ func FromPot(b []byte) (model.Messages, error) {
+			return fmt.Errorf("write %s: %w", tag, err)
 		}
 
-		fuzzy := strings.Contains(node.Flag, "fuzzy")
+		return nil
+	}
+
+	if _, err = fmt.Fprintf(b, "%s \"\"\n", tag); err != nil {
+		return fmt.Errorf("write %s: %w", tag, err)
+	}
+
+	if err = writeMultiline(b, tag, lines); err != nil {
+		return fmt.Errorf("write multiline: %w", err)
+	}
+
+	return nil
+}
 
-		message := model.Message{
-			ID:          node.MsgId,
-			Description: strings.Join(node.ExtractedComment, "\n"),
-			Fuzzy:       fuzzy,
+func writePluralForms(b *bytes.Buffer, tag poTag, str string) error {
+	nodes, err := messageformat.Parse(str)
+	if err != nil {
+		return fmt.Errorf("parse message: %w", err)
+	}
+
+	for _, node := range nodes {
+		nodeMatch, ok := node.(messageformat.NodeMatch)
+		if !ok {
+			return errors.New("convert node to messageformat.NodeMatch")
+		}
+
+		if err = writeVariants(b, tag, nodeMatch); err != nil {
+			return fmt.Errorf("write variants: %w", err)
 		}
+	}
 
-		if len(node.MsgStr) > 0 {
-			if strings.HasSuffix(node.MsgStr[0], "\\n") {
-				node.MsgStr[0] = strings.ReplaceAll(node.MsgStr[0], "\\n", "\n")
+	return nil
+}
+
+func writeVariants(b *bytes.Buffer, tag poTag, nodeMatch messageformat.NodeMatch) error {
+	for i, variant := range nodeMatch.Variants {
+		if _, err := fmt.Fprintf(b, "msgstr[%d] ", i); err != nil {
+			return fmt.Errorf("write plural msgstr: %w", err)
+		}
+
+		var txt strings.Builder
+
+		for _, msg := range variant.Message {
+			switch node := msg.(type) {
+			case messageformat.NodeText:
+				txt.WriteString(node.Text)
+			case messageformat.NodeVariable:
+				txt.WriteString("%d")
+			default:
+				return errors.New("unknown node type")
 			}
+		}
 
-			message.Message = node.MsgStr[0]
+		lines, err := getPoTagLines(txt.String(), tag)
+		if err != nil {
+			return fmt.Errorf("get po tag lines: %w", err)
 		}
 
-		messages = append(messages, message)
+		if len(lines) == 1 {
+			if _, err = fmt.Fprintf(b, "\"%s\"\n", lines[0]); err != nil {
+				return fmt.Errorf("write %s: %w", tag, err)
+			}
+
+			continue
+		}
+
+		if _, err = fmt.Fprintf(b, "\"\"\n"); err != nil {
+			return fmt.Errorf("write %s: %w", tag, err)
+		}
+
+		if err = writeMultiline(b, tag, lines); err != nil {
+			return fmt.Errorf("write multiline: %w", err)
+		}
 	}
 
-	return model.Messages{
-		Language: po.Header.Language,
-		Messages: messages,
-	}, nil
+	return nil
 }
 
-func writeToPoTag(b *bytes.Buffer, tag poTag, str string) error {
+func writeMultiline(b *bytes.Buffer, tag poTag, lines []string) error {
+	for _, line := range lines {
+		if !strings.HasSuffix(line, "\\n") {
+			line += "\\n"
+		}
+
+		if _, err := fmt.Fprint(b, "\""+line+"\""+"\n"); err != nil {
+			return fmt.Errorf("write %s: %w", tag, err)
+		}
+	}
+
+	return nil
+}
+
+func getPoTagLines(str string, tag poTag) ([]string, error) {
 	encodedStr, err := json.Marshal(str) // use JSON encoding to escape special characters
 	if err != nil {
```

Comment:
how about [strconv.Qoute](https://pkg.go.dev/strconv#Quote) to remove special characters and minimize overhead
```suggestion
func getPoTagLines(str string, tag poTag) ([]string, error) {
    encodedStr := strconv.Quote(str)
```